### PR TITLE
feat: Add Timestamp on watch

### PR DESCRIPTION
**Pull request by ev1stensberg**
Adds file stamp on watch from #21.

### CODE REVIEW

File: bin/process-options.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ module.exports = function processOptions(yargs, argv) {
 	}
 
 	if(argv.progress) {
-		var ProgressPlugin = require("../lib/ProgressPlugin");
+		var ProgressPlugin = require("../node_modules/webpack/lib/ProgressPlugin");
```
Comment (pksjce) [inline bin/process-options.js]: Won't `webpack/lib/ProgressPlugin` work?

Comment (ev1stensberg) [inline bin/process-options.js]: Any change I did? Do you have the console log from it?

Comment (ev1stensberg) [inline bin/process-options.js]: Can change it later, doing another PR within this, so don't merge yet.

**Comment by ev1stensberg** [comment]
Don't look at this PR, it's a trap

**Pull request closed without merging**